config.cfg

```diff
@@ -0,0 +1,19 @@
+
+# Configuation file
+[app]
+DEBUG = True
+# This key will be regenerated and hidden at the end of this project
+SECRET_KEY = A0Zr98j/3yX R~XHH!jmN]LWX/,?RT
+SQLALCHEMY_DATABASE_URI = postgres://localhost/lemur
+
+[login_manager]
+SESSION_PROTECTION = strong
+LOGIN_VIEW = login
+
+[logger]
+FILEPATH = error_log/errors.log
+LEVEL = WARNING
+
+[class_time_info]
+CLASS_TIME = 16fall
+CURRENT_TIME = 16fall
```

Comment:
You have no guarantee that they are making a class for the current year. You should simply have the prof enter information, with a default to the current year and season.